logic/src/androidMain/kotlin/com/wire/kalium/logic/feature/call/scenario/OnParticipantListChanged.kt

```diff
@@ -34,9 +41,13 @@ class OnParticipantListChanged(
 
         val participantsChange = Json.decodeFromString<CallParticipants>(data)
         callingScope.launch {
+            val memberList: List<Member> = conversationRepository
+                .observeConversationMembers(remoteConversationIdString.parseIntoQualifiedID())
+                .first()
+
```

Comment:
I am wondering why getting member list ? `participantsChange` already have list of participants with their ids

Reply:
The problem here is that the member list ids come without domain when non federated. So we need these ids fully qualified later on to fetch user details, ie: name, avatar, etc. Because the endpoint to fetch `userInfo` doesn't support not sending a domain :(